build/dependencies.yaml

```diff
@@ -80,7 +80,7 @@ dependencies:
       match: configs\[Etcd\] = Config{list\.GcEtcdRegistry, "etcd", "\d+\.\d+.\d+(-(alpha|beta|rc).\d+)?(-\d+)?"}
 
   - name: "etcd-image"
-    version: 3.5.17
+    version: 3.5.18
```

Comment:
maybe we should wait for 3.5.19

REF: https://github.com/etcd-io/etcd/issues/19406

Reply:
/hold